### PR TITLE
adjust consul status

### PR DESCRIPTION
**Pull request by mshima**
---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/client/templates/angular/src/main/webapp/app/admin/gateway/gateway.component.ts.ejs [hunk 1]
```diff
@@ -40,17 +40,19 @@ export class GatewayComponent implements OnInit {
...
       <%_ if (serviceDiscoveryType === 'consul') { _%>
-      gatewayRoutes.serviceInstances = gatewayRoutes.serviceInstances.map(serviceInstance => {
-        if (serviceInstance.healthService && serviceInstance.healthService.checks) {
-          if (serviceInstance.healthService.checks.filter(
-            check => check.status === 'PASSING'
-          ).length === serviceInstance.healthService.checks.length) {
-            serviceInstance.instanceInfo.status = 'UP'
-          } else {
-            serviceInstance.instanceInfo.status = 'DOWN'
+      gatewayRoutes.map(route => {
```
Comment (mshima) [inline generators/client/templates/angular/src/main/webapp/app/admin/gateway/gateway.component.ts.ejs:43]:
```suggestion
      gatewayRoutes.forEach(route => {
```

File: generators/client/templates/vue/src/main/webapp/app/admin/gateway/gateway.component.ts.ejs [hunk 2]
```diff
@@ -17,17 +17,19 @@ export default class <%=jhiPrefixCapitalized%>GatewayComponent extends Vue {
...
       <%_ if (serviceDiscoveryType === 'consul') { _%>
-      this.gatewayRoutes.serviceInstances = this.gatewayRoutes.serviceInstances.map(serviceInstance => {
-        if (serviceInstance.healthService && serviceInstance.healthService.checks) {
-          if (serviceInstance.healthService.checks.filter(
-            check => check.status === 'PASSING'
-          ).length === serviceInstance.healthService.checks.length) {
-            serviceInstance.instanceInfo.status = 'UP'
-          } else {
-            serviceInstance.instanceInfo.status = 'DOWN'
+      this.gatewayRoutes.map(route => {
```
Comment (mshima) [inline generators/client/templates/vue/src/main/webapp/app/admin/gateway/gateway.component.ts.ejs:20]:
```suggestion
      this.gatewayRoutes.forEach(route => {
```

**Comment by deepu105** [comment]
was this code already there. I thought I wrote it for the consul as default PR :thinking:

**Comment by mshima** [comment]
You may have mistakenly pushed to main https://github.com/jhipster/generator-jhipster/commit/cf49603fb5b11df79f5dc1f3e467e0064a498d19.

**Comment by deepu105** [comment]
I think the commit got in by accident. I'll revert this and do a new release. So I'll close this PR

**Pull request closed without merging**